kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -251,11 +255,14 @@ private Config(Boolean autoConfigure) {
    * You can also reuse this object to build your own {@link Config} object
    * without any auto configuration like this:
    *
-   * <pre>{@code
-   * Config configFromBuilder = new ConfigBuilder(Config.empty())
-   *                                // ...
-   *                               .build();
-   * }</pre>
+   * <pre>
+   * {
+   *   &#64;code
+   *   Config configFromBuilder = new ConfigBuilder(Config.empty())
+   *       // ...
+   *       .build();
+   * }
+   * </pre>
```

Comment:
So my assumption is incorrect... 
https://github.com/fabric8io/kubernetes-client/pull/3913#discussion_r820702410

We need to change the formatter configuration to prevent this.

Reply:
The correct option is `org.eclipse.jdt.core.formatter.comment.format_source_code`, which is working now: https://github.com/fabric8io/kubernetes-client/runs/5462546383?check_suite_focus=true